README.md

```diff
@@ -0,0 +1,112 @@
+# Secure AWS Instance Profiles on DC/OS
+Security is always an important topic in today's distributed systems. With DC/OS Enterprise we offer a feature called DC/OS Secrets. With Secrets it is possible to inject secure information like passwords or cryptographic keys into your application. No other Application is able to read or change this information and with DC/OS IAM you can also strip down the group of users having access to this information.
+
+## The usual workflow
+Let's assume you have an application that wants to access AWS resources, like a S3 bucket. With Secrets you can easily create an IAM user and assign it a Policy which is able to access your particular bucket. You take the ACCESS_KEY and add it to your Marathon specification and store the SECRET_ACCESS_KEY into a DC/OS Secret in your default vault which you also specify in your marathon application. This is absolutely not bad but it means you have to rotate these credentials on a regular basis and therefore you need to update your application from time to time.
```

Comment:
Let's assume you have an application that wants to access AWS resources, like an S3 bucket. With Secrets you can easily create an IAM user and assign it a Policy which is able to access your particular bucket. You take the ACCESS_KEY and add it to your Marathon specification and store the SECRET_ACCESS_KEY into a DC/OS Secret in your default vault which you also specify in your Marathon application. This is not bad, but it means you must rotate these credentials on a regular basis and therefore you need to update your application from time to time.